simulation/traffic_simulator/test/src/traffic_lights/test_traffic_light.cpp

```diff
@@ -723,39 +726,68 @@ TEST(TrafficLight, Bulb_trafficLightMessageConversion)
   }
 }
 
-/**
- * @note Test basic functionality. Test function behavior when called with invalid name.
- */
-TEST(TrafficLight, Color_make_wrong)
+class TrafficLightTest : public testing::Test
 {
-  EXPECT_THROW(traffic_simulator::TrafficLight::Color::make("wrong_color"), common::SyntaxError);
-}
+protected:
+  TrafficLightTest()
+  : map_manager(
+      ament_index_cpp::get_package_share_directory("traffic_simulator") + "/map/lanelet2_map.osm",
+      geographic_msgs::build<geographic_msgs::msg::GeoPoint>()
+        .latitude(35.61836750154)
+        .longitude(139.78066608243)
+        .altitude(0.0))
+  {
+  }
+  hdmap_utils::HdMapUtils map_manager;
+};
 
 /**
- * @note Test basic functionality. Test function behavior when called with invalid name. 
+ * @note test if function correctly determines if a given bulb
+ * is in the bulbs vector given a Color, Status, Shape triple.
  */
-TEST(TrafficLight, Shape_make_wrong)
+TEST_F(TrafficLightTest, contains_colorStatusShape)
 {
-  EXPECT_THROW(traffic_simulator::TrafficLight::Shape::make("wrong_shape"), common::SyntaxError);
-}
+  using TrafficLight = traffic_simulator::TrafficLight;
+  using Color = TrafficLight::Color;
+  using Status = TrafficLight::Status;
+  using Shape = TrafficLight::Shape;
```

Comment:
I think this should be moved to the fixture class

Reply:
I will just move usings to the top of the file